src/main/kotlin/org/malachite/estella/aplication/domain/ApplicationPayloads.kt

```diff
@@ -18,46 +19,46 @@ import java.util.*
 //TODO: This solution with separate payloads is temporary
```

Comment:
Pytanie, czy ono w takim razie dalej jest temporary? XD

Reply:
To jest do przegadania ale nie mam obecnie pomysłu jak to inaczej zrobić

Reply:
To na pewno, po prostu zapomniałem, że coś takiego tu było XD - teoretycznie można patrzeć na jwt i z niego pobierać jakieś informacje. Co prawda nie pamiętam, co jest w tamtych payloadach, a jest trochę za późno, żeby jeszcze teraz na to patrzeć, także poczeka to do half weekly XD